<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einrichten</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>An einer Wand ausrichten</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>248</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Bitte ziehlen Sie mit dem LASER auf die Eckpunkte&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;der Wand und klicken Sie auf den&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;entsprechenden Button.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Manuelle Bedienung</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="Btn_left">
           <property name="text">
            <string>links</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="Btn_up">
           <property name="text">
            <string>oben</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="Btn_right">
           <property name="text">
            <string>rechts</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="Btn_down">
           <property name="text">
            <string>unten</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="Btn_ro">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>rechts oben</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="Btn_lo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>links oben</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Abstand zur Wand</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinBx_abstand">
          <property name="suffix">
           <string>cm</string>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_distance">
          <property name="text">
           <string>Ultraschall</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="Btn_lu">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>links unten</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>248</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="Btn_ru">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>rechts unten</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_cancel">
       <property name="text">
        <string>Abbrechen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_next">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Weiter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBx_abstand</tabstop>
  <tabstop>Btn_left</tabstop>
  <tabstop>Btn_right</tabstop>
  <tabstop>Btn_up</tabstop>
  <tabstop>Btn_down</tabstop>
  <tabstop>Btn_lo</tabstop>
  <tabstop>Btn_ro</tabstop>
  <tabstop>Btn_cancel</tabstop>
  <tabstop>Btn_next</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_next</sender>
   <signal>clicked()</signal>
   <receiver>settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_lo</sender>
   <signal>toggled(bool)</signal>
   <receiver>Btn_ro</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_ro</sender>
   <signal>toggled(bool)</signal>
   <receiver>Btn_lo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_ro</sender>
   <signal>toggled(bool)</signal>
   <receiver>Btn_ru</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_ru</sender>
   <signal>toggled(bool)</signal>
   <receiver>Btn_ro</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_lu</sender>
   <signal>toggled(bool)</signal>
   <receiver>Btn_ru</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_ru</sender>
   <signal>toggled(bool)</signal>
   <receiver>Btn_lu</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_lu</sender>
   <signal>toggled(bool)</signal>
   <receiver>Btn_lo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_lo</sender>
   <signal>toggled(bool)</signal>
   <receiver>Btn_lu</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
